tests/test_routing_lievre_tutorial.py

```diff
@@ -234,3 +236,235 @@ def test_lievre_tutorial(self):
+            SoilProfilesCommand.Record("LAKE", ("AQUIFER",), (5,)),
+            SoilProfilesCommand.Record("Soil_Land_HRU", ("AQUIFER",), (5,)),
+        ]
+        model.config.rvp.vegetation_classes = [
+            VegetationClassesCommand.Record("Veg_Land_HRU", 25, 5.0, 5.0),
+            VegetationClassesCommand.Record("WATER", 0, 0, 0),
+        ]
+        model.config.rvp.land_use_classes = [
+            LandUseClassesCommand.Record("Landuse_Land_HRU", 0, 1),
+            LandUseClassesCommand.Record("WATER", 0, 0),
+        ]
+        model.config.rvp.channel_profiles = channel_profiles
+
+        #######
+        # RVT #
+        #######
+
+        streaminputs_extractor = RoutingProductGridWeightExtractor(
+            vic_streaminputs_nc_path, routing_product_shp_path
+        )
+
+        temperatures_extractor = RoutingProductGridWeightExtractor(
+            vic_temperatures_nc_path, routing_product_shp_path
+        )
+
+        model.config.rvt.set_nc_variables(
+            [
+                dict(
+                    name="StreamInputs",
+                    data_type="PRECIP",
+                    file_name_nc=vic_streaminputs_nc_path.name,
+                    var_name_nc="Streaminputs",
+                    dim_names_nc=("lon_dim", "lat_dim", "time"),
+                    scale=4.0,
+                    offset=0,
+                    grid_weights=streaminputs_extractor.extract(),
+                ),
+                dict(
+                    name="AverageTemp",
+                    data_type="TEMP_AVE",
+                    file_name_nc=vic_temperatures_nc_path.name,
+                    var_name_nc="Avg_temp",
+                    dim_names_nc=("lon_dim", "lat_dim", "time"),
+                    grid_weights=temperatures_extractor.extract(),
+                ),
+                dict(
+                    is_observation=True,
+                    data_type="HYDROGRAPH",
+                    subbasin_id=gauged_sb.subbasin_id,
+                    units="m3/s",
+                    file_name_nc=observation_data_nc_path.name,
+                    var_name_nc="Q",
+                    dim_names_nc=("nstations", "time"),
+                    index=1,  # StationIdx
+                ),
+            ]
+        )
+
+        #############
+        # Run model #
+        #############
+
+        model(
+            [
+                vic_streaminputs_nc_path,
+                vic_temperatures_nc_path,
+                observation_data_nc_path,
+            ],
+        )
+
+        ##########
+        # Verify #
+        ##########
+
+        assert len(model.hydrograph.time) == (end - start).days + 1
+
+        assert model.hydrograph.basin_name.item() == gauged_sb.name
+
+        csv_lines = model.outputs["diagnostics"].read_text().split("\n")
+
+        assert csv_lines[1].split(",")[:-1] == [
+            "HYDROGRAPH_ALL",
+            observation_data_nc_path.name,
+            "0.253988",
```

Comment:
Shall we add comments here that these are:
```
"NASH_SUTCLIFFE",
"PCT_BIAS",
"KLING_GUPTA"
```

Reply:
done